pkg/annotation/capa.go

```diff
@@ -0,0 +1,23 @@
+package annotation
```

Comment:
As we're adding CAPA specific annotations, should we also add the network-topology ones to this?
* `network-topology.giantswarm.io/mode`
* `network-topology.giantswarm.io/transit-gateway`
* `network-topology.giantswarm.io/prefix-list`